docs/spec/sanitations.bal

```diff
@@ -112,6 +148,30 @@ function removeRequestBodyFromHttpGet(Specification spec) returns Specification
     return spec;
 }
 
+function removeReqBodyFromBodyNonRequiredOps(Specification spec) returns Specification {
+    map<Path> paths = spec.paths;
+    foreach var ['key, value] in paths.entries() {
+        Delete? delete = value.delete;
+        if delete is () {
+            continue;
+        }
+
+        string? operationId = delete.operationId;
+        if operationId is () {
+            continue;
+        }
+
+        if REQUEST_BODY_NON_REQUIRED_OPS.indexOf(operationId) is () {
+            continue;
+        }
+
+        _ = delete.removeIfHasKey(REQUEST_BODY);
+        value.delete = delete;
+        paths['key] = value;
```

Comment:
can we write this way?
```suggestion
Delete? delete = value.delete;
string? operationId = delete?.operationId;
if delete is Delete && operationId is string && REQUEST_BODY_NON_REQUIRED_OPS.indexOf(operationId) != () {
    _ = delete.removeIfHasKey("REQUEST_BODY");
    value.delete = delete;
    paths['key] = value;
}
```

Reply:
I would rather keep it as it is cos it is more clearer that way rather than a complex if-condition.

Reply:
+1